src/string-util/string-util.ts

```diff
@@ -3,6 +3,11 @@ import type { Tag } from './string-util.interface';
 const DOMESTIC_PHONE_NUMBER_REGEXP = /^0[1,7]\d{9}$/;
 
 export namespace StringUtil {
+  export function getNonce(nonceLength: number, nonceEncoding: number): string {
+    const substringStartAt = -nonceLength;
+    return Math.random().toString(nonceEncoding).substr(substringStartAt);
```

Comment:
이정도는 굳이 할당 안해줘도 되겠습니다
```suggestion
    return Math.random().toString(nonceEncoding).substr(-nonceLength);
```

Reply:
아.. 그러네요! 수정했습니다